benchmark_schema.py

```diff
@@ -26,6 +26,8 @@ class TestRun(Base):
     distributed_version = Column(String, nullable=True)
     python_version = Column(String, nullable=True)
     platform = Column(String, nullable=True)
+    cluster_name = Column(String, nullable=True)
+    cluster_id = Column(Integer, nullable=True)
```

Comment:
Not that @ncclementi is doing the same thing in #401. Also, this will require an alembic migration

Reply:
PR for reference, https://github.com/coiled/coiled-runtime/pull/433

Reply:
I've reverted my version